### ISSUE TITLE
[Feature Request] Adjustable window layout

### ISSUE THREAD

**Issue by random-yang** [Slowlydev/f1-dash#77]
It would be great if these parts can be adjust.
<img width="1727" alt="image" src="https://github.com/Slowlydev/f1-dash/assets/25839518/34f4cd33-275e-4070-9fc5-6b6ec1c5b893">

**Comment by Slowlydev** [comment]
Yeah I also thought about it...
In v2 I am planning to implement a future where the user can open the map, leaderboard, weather and more in new windows and customise their setup across multiple screens. Would that cover your future request? Or how would you like to see this being implemented?

**Comment by Aarekaz** [comment]
Yeah, I think that is what he means and I also had something similar in my mind. If you want help with UI, let me know I can help design  figure out something so you don't have to think too much.

**Comment by Aarekaz** [comment]
Yup! I can help you design it out simply. Can I message you in discord/iMessage somewhere?

**Comment by Aarekaz** [comment]
Awesome, just sent you a request in discord. I have the same username.  I can help setup discord/manage it as well as help in the general non main dev stuff. Chat with you there.

**Comment by slowlydev** [comment]
@random-yang how are you liking the new v2 windows, does this close this Issue/Feature Request?

**Comment by random-yang** [comment]
> @random-yang how are you liking the new v2 windows, does this close this Issue/Feature Request?

It's much better! Great job.
